lemur/plugins/lemur_aws/plugin.py

```diff
@@ -306,17 +306,13 @@ def get_load_balancers(self, options, **kwargs):
             regions = "".join(regions.split()).split(",")
 
         for region in regions:
-            try:
-                elbs = elb.get_all_elbs(account_number=account_number, region=region)
-                current_app.logger.info({
-                    "message": "Describing classic load balancers",
-                    "account_number": account_number,
-                    "region": region,
-                    "number_of_load_balancers": len(elbs)
-                })
-            except Exception as e:  # noqa
```

Comment:
is my understanding correct, that we are removing the try and catch, to handle the exception all the way through ` sync_endpoints` 

https://github.com/Netflix/lemur/blob/a69c9e01e846a4fb86f8351b1164608d19b7f344/lemur/sources/service.py#L75

in `sync`?

This would be a failure at the account level, which would break the entire syn cert, sync endpoints, and expire endpoints. I believe this is the intended behavior to fail close.

Reply:
Your understanding is correct, we want exceptions here to abort all syncing for the source before we do any DB writes.